src/encoded/upgrade/dataset.py

```diff
@@ -364,3 +364,10 @@ def dataset_16_17(value, system):
 def dataset_17_18(value, system):
     if value['status'] == "started":
         value['status'] = "in progress"
+
+
+@upgrade_step('experiment', '18', '19')
+def dataset_18_19(value, system):
```

Comment:
I know I didn't do this for the last upgrade I wrote but we could clean this up to use all single quotes instead of a mix of single and double quotes.

Reply:
fixed in the upgrade I have added